apps/web/components/getting-started/components/ConnectedCalendarItem.tsx

```diff
@@ -14,6 +14,7 @@ interface IConnectedCalendarItem {
     userId?: number | undefined;
     integration?: string | undefined;
     externalId: string;
+    domainWideDelegationCredentialId?: string | null;
```

Comment:
I think we should keep it a required prop with possible value being null as that would ensure that we never forget to pass the prop.
@Udit-takkar

Reply:
Handled it in my PR https://github.com/calcom/cal.com/pull/17958/files